src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -103,8 +107,80 @@ export const ImageOutputList = () => {
   );
 };
 export const FSCList = () => {
+  //const isManual =
+  //  getState()?.values?.['file-system-config-radio'] === 'manual';
+  //const partitions = getState()?.values?.['file-system-configuration'];
```

Comment:
Do we want to just delete this commented code? Since it's calling a DDF function, I can't think of a good reason to keep it.

Reply:
I suppose this is part of the skeleton for FSC... To be honest, I would prefer not adding a skeleton in the form of comments. If we could figure out a way to do this without commenting the code, that would be better.

Can we split out the ReleaseLifecycle dates fix and the FSC list stuff into separate commits? Maybe for this PR we can just have the ReleaseLifeCycle date fix and then we can worry about the FSC review step later, once we have the actual step merged.

Reply:
Fair enough, I've updated the PR to fix the ReleaseLifecycle dates only :+1: